libraries/lib-wave-track/WaveClip.h

```diff
@@ -220,6 +221,14 @@ class WAVE_TRACK_API WaveClip final : public XMLTagHandler
    //! @returns Number of samples within t0 and t1 if t1 > t0, 0 otherwise
    sampleCount CountSamples(double t0, double t1) const;
 
+   /*
+    * @brief Request up to `length` samples. The actual number of samples
+    * available from the returned view is queried through
+    * `AudioSegmentSampleView::GetSampleCount()`.
+    */
```

Comment:
You need more commentary to explain `ii` and state the precondition on its legal values.

Then, it is understood that the burden of a guarantee of the correctness of the assumption lies not in the function but in the places that use it.

Reply:
```
   /*!
    @param ii identifies the channel
    @pre `ii < GetWidth()`
    */
```

Reply:
Thanks.